shared/src/crypto/hash.rs

```diff
@@ -0,0 +1,14 @@
+use multihash::derive::Multihash;
+use multihash::{Blake2b256, Keccak256, Sha2_256};
+
+#[derive(Clone, Copy, Debug, Eq, Multihash, PartialEq)]
+#[mh(alloc_size = 32)]
+pub enum FvmHashCode {
```

Comment:
Let's call this enum `SupportedHashes` -- that's the role it plays in the system. `FvmHashCode` is a poor name because it carries little information besides what can be inferred from the context:

1. We know we're talking about the fvm because this is inside the `fvm_shared` crate, so `Fvm` is superfluous.
2. We can easily infer this is a code for a hash function from everything else.